### DISCUSSION TITLE
React native wallet connect

### DISCUSSION THREAD

**Discussion by safaksonmez** [WalletConnect/walletconnect-monorepo#814] [Developer Support]
Hello I am trying to connect with metamask via walletconnect provider.I can connect with ios but when ı try to connect with android it doesnt show anything at all
![image](https://user-images.githubusercontent.com/40964682/155321521-61da361e-3aea-40cb-ac3e-3734883f3708.png)
![image](https://user-images.githubusercontent.com/40964682/155321538-c81ba0ea-bbef-4b26-bba3-ff2811b97f04.png)
these are ios it works fine but on android connect button doesnt do anything at all
![image](https://user-images.githubusercontent.com/40964682/155321624-d0357aed-bc11-432b-91b7-12bff6b74a86.png)
![image](https://user-images.githubusercontent.com/40964682/155321648-26e9b24d-c704-404b-8a0a-5b602fa08ee1.png)

**Comment by Victorvikson1996** [comment]
Try and write a function to kill connection

**Comment by daqing-li** [comment]
Can you please share the code base to mobile52521@gmail.com?
Also please let me know the OS env as well.
I will have a look about that. that works well on Android for me,
